heidegger_index/views.py

```diff
@@ -2,6 +2,9 @@
 from django.views.generic.detail import DetailView
 
 from heidegger_index.models import Lemma, Work, PageReference
+from heidegger_index.settings import BASE_DIR
```

Comment:
Als je waarden uit de settings gebruikt is het [gebruikelijk](https://docs.djangoproject.com/en/4.0/topics/settings/#using-settings-in-python-code) om dat zo te doen:

```python
from django.conf import settings
```

En dan `settings.BASE_DIR` gebruiken waar nodig.

Reply:
Bedankt voor de heads-up! Ik heb het veranderd!